server/controller/src/main/resources/application.yaml

```diff
@@ -56,6 +56,7 @@ sw:
         log-cache-size: ${DATASET_CONSUMPTION_LOG_CACHE_SIZE:1000}
         log-cache-timeout: ${DATASET_CONSUMPTION_LOG_CACHE_TIMEOUT:24h}
         lock-wait-seconds: ${DATASET_CONSUMPTION_LOCK_WAIT_SECONDS:10}
+        deal-error-session-interval: ${DATASET_CONSUMPTION_DEAL_ERROR_SESSION_INTERNAL:1000}
```

Comment:
1000 to 1s will have better readability? and will give the reader an explicit time format example.

Reply:
done